Commands/settings.py

```diff
@@ -0,0 +1,99 @@
+import discord
+import asyncio
+
+from discord.ext import commands
+from .GuildData import get_guild_data, save_configs
+
+
+class Settings(commands.Cog):
+    def __init__(self, bot):
+        self.bot = bot
+
+    @commands.command(name="add_admin", brief="**admin-only** \n\u2003 Add a bot admin", usage="<mentioned user>", help="*admin-only* \u2003 Add a bot admin. The user will be given access to the bot commands that are labelled **admin-only**. The user will not be given the role of server admin. \n\n<mention user> \u2003 Mention the user that you want to make a bot admin.")
+    async def add_admin(self, ctx, mention):
+        guild_data = await get_guild_data(ctx.message.guild.id)
+
+        if not (ctx.message.author.guild_permissions.administrator or ctx.author.id in guild_data.bot_admins):
+            # if the user is not a bot admin or server admin
+            await ctx.send("https://gph.is/g/4w8PDNj")
```

Comment:
Should there not be a return after this await send here was well?
Better yet, change into desired case in the if (so admin) and handle this in the else after.

Reply:
In my opinion the order of the if statements is clearer now, because there are two separate if statements. First to check permissions and input and then the second to handle the command.